ctk/idp/coverage/Core.md

```diff
@@ -0,0 +1,171 @@
+# Core Specification
+This file tracks the testing coverage of the SAML Core specification.
+
+### Legend
+```diff
++ Fully tested
+- Ignored
+Unmarked sections need attention
+```
+
+### Table of Contents
+```diff
+ 1 Introduction
+-	1.1 Notation
+-	1.2 Schema Organization and Namespaces
+	1.3 Common Data Types
+		1.3.1 String Values
++		1.3.2 URI Values
+		1.3.3 Time Values
+		1.3.4 ID and ID Reference Values
+2 SAML Assertions
+-	2.1 Schema Header and Namespace Declarations
+	2.2 Name Identifiers
+-		2.2.1 Element <BaseID>
+-		2.2.2 Complex Type NameIDType
+-		2.2.3 Element <NameID>
+		2.2.4 Element <EncryptedID>
+-		2.2.5 Element <Issuer>
+	2.3 Assertions
+-		2.3.1 Element <AssertionIDRef>
+-		2.3.2 Element <AssertionURIRef>
+		2.3.3 Element <Assertion>
+		2.3.4 Element <EncryptedAssertion>
+	2.4 Subjects
+		2.4.1 Element <Subject>
++			2.4.1.1 Element <SubjectConfirmation>
+			2.4.1.2 Element <SubjectConfirmationData>
+			2.4.1.3 Complex Type KeyInfoConfirmationDataType
+-			2.4.1.4 Example of a Key-Confirmed <Subject>
+	2.5 Conditions
+		2.5.1 Element <Conditions>
+			2.5.1.1 General Processing Rules
++			2.5.1.2 Attributes NotBefore and NotOnOrAfter
+-			2.5.1.3 Element <Condition>
+			2.5.1.4 Elements <AudienceRestriction> and <Audience>
+			2.5.1.5 Element <OneTimeUse>
+			2.5.1.6 Element <ProxyRestriction>
+-	2.6 Advice
+-		2.6.1 Element <Advice>
+	2.7 Statements
+-		2.7.1 Element <Statement>
++		2.7.2 Element <AuthnStatement>
+-			2.7.2.1 Element <SubjectLocality>
+-			2.7.2.2 Element <AuthnContext>
++		2.7.3 Element <AttributeStatement>
+			2.7.3.1 Element <Attribute>
++				2.7.3.1.1 Element <AttributeValue>
+			2.7.3.2 Element <EncryptedAttribute>
+		2.7.4 Element <AuthzDecisionStatement>
+-			2.7.4.1 Simple Type DecisionType
++			2.7.4.2 Element <Action>
+-			2.7.4.3 Element <Evidence>
+3 SAML Protocols
+-	3.1 Schema Header and Namespace Declarations
+	3.2 Requests and Responses
+		3.2.1 Complex Type RequestAbstractType
+		3.2.2 Complex Type StatusResponseType
++			3.2.2.1 Element <Status>
++			3.2.2.2 Element <StatusCode>
+-			3.2.2.3 Element <StatusMessage>
+-			3.2.2.4 Element <StatusDetail>
+	3.3 Assertion Query and Request Protocol
+-		3.3.1 Element <AssertionIDRequest>
+		3.3.2 Queries
+-			3.3.2.1 Element <SubjectQuery>
+			3.3.2.2 Element <AuthnQuery>
+				3.3.2.2.1 Element <RequestedAuthnContext>
+			3.3.2.3 Element <AttributeQuery>
+			3.3.2.4 Element <AuthzDecisionQuery>
+-		3.3.3 Element <Response>
+		3.3.4 Processing Rules
++	3.4 Authentication Request Protocol
+		3.4.1 Element <AuthnRequest>
+			3.4.1.1 Element <NameIDPolicy>
+-			3.4.1.2 Element <Scoping>
+			3.4.1.3 Element <IDPList>
++				3.4.1.3.1 Element <IDPEntry>
+			3.4.1.4 Processing Rules
+-			3.4.1.5 Proxying
+				3.4.1.5.1 Proxying Processing Rules
+	3.5 Artifact Resolution Protocol
++		3.5.1 Element <ArtifactResolve>
+		3.5.2 Element <ArtifactResponse>
+		3.5.3 Processing Rules
+	3.6 Name Identifier Management Protocol
```

Comment:
We can ignore:
- `3.6`, `3.6.1`, `3.6.2`, `3.6.3`
- `3.7`, `3.7.2`, `3.7.3.1`, `3.7.3.2`
- `3.8`, `3.8.1`, `3.8.2`, `3.8.3`